brave/browser-laptop
# Add notification to suggest users use Brave Payments

Fix #3257 (Finishes the remaining task)

- Record firstRunTimestamp into session-state
- 10 days from firstRunTimestamp, show notification suggesting user tries Payments
- If the user enables Payments they won't see the notification



Auditors: @bsclifton @diracdeltas

Test Plan:

1. Edit `ledger.js` to reduce showNotifications delay interval to 3 * msecs.second
2. Run Brave with an existing profile
3. Confirm there's no notification
4. Close Brave
5. Update userData session-store-1 firstRunTimestamp to be way in the past e.g. from '14..' to '13..' (~/Library/Application\ Support/session-store-1)
6. Open Brave
7. Confirm notification
